### PR TITLE
Feature/parking lot route

### PR DESCRIPTION
**Pull request by anguyen698**
# Administrative Info
Make sure your branch name conforms to: `feature/hotfix/.../describe_your_branch_with_underscores

### Changes
What changes did you make?
- Added GET /parking_lot route
- Used get() to retrieve parking_lot info

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tested my program thoroughly
- [ ] Did you add/sub fields in the environment file? if so please update .env_sample

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Code change and  localhost:3000 working |
| --- |
| get-parking-lot-screenshot|

### CODE REVIEW

**Review by Eldar827 — APPROVED** [review]
lgtm

**Review by samuelng678 — APPROVED** [review]
lgtm

File: routes/parking_lot.js [hunk 1]
```diff
@@ -3,6 +3,11 @@ var router = express.Router();
 
 const Lots = require("../database/model/lotModel");
```
Comment (YuyusanK) [inline routes/parking_lot.js:4]: Lots should be named lot_model

File: routes/parking_lot.js [hunk 2]
```diff
@@ -3,6 +3,11 @@ var router = express.Router();
...
 
+/* GET parking_lot route */
+router.get("/parking_lot", (req, res) => {
+    Lots.get();
```
Comment (YuyusanK) [inline routes/parking_lot.js:8]: Save the returned values in a variable.

**Review by YuyusanK — CHANGES_REQUESTED** [review]
Please fix those sections

**Review by YuyusanK — APPROVED** [review]
lgtm

**Pull request merged**